src/net/sourceforge/kolmafia/persistence/AdventureSpentDatabase.java

```diff
@@ -132,17 +136,19 @@ public static int getTurns(final String loc) {
   }
 
   public static void serialize() {
-    File file =
-        new File(KoLConstants.DATA_LOCATION, KoLCharacter.baseUserName() + "_" + "turns.ser");
-
-    try {
-      FileOutputStream fileOut = new FileOutputStream(file);
-      ObjectOutputStream out = new ObjectOutputStream(fileOut);
-
-      out.writeObject(AdventureSpentDatabase.TURNS);
-      out.close();
-    } catch (IOException e) {
-      e.printStackTrace();
+    if (allowSerializationWrite) {
```

Comment:
would probably prefer to see this as:
```
if (!allowSerializationWrite) return;
```

to make it obvious the method does nothing else

Reply:
Not my natural idiom but OK.  Long long ago in a structured language no one remembers the coding standard was one return per function.  That habit persists even when it approaches being a Bad Habit :-)

Reply:
Delphi?

I think this pattern is called a "guard clause" nowadays. It tends to be preferred from what I've seen.